src/Devices/PHProbe.cpp

```diff
@@ -64,6 +66,14 @@ void PHProbe::serialEvent1() {
       if (isdigit(string[0])) {  // if the first character in the string is a digit
         // convert the string to a floating point number so it can be evaluated by the Arduino
         value = string.toFloat();
+        // we have seen situations where the CO2 bubbler stays on and drives the pH down
+        if (value < 7.0) {  // hang so as to trigger the watchdog timer reset
```

Comment:
While this could be hard coded, this seems like a threshold that should be user defined. Maybe we open an issue for customization.